platform/engineering/frontend/platform-assignments.md

```diff
@@ -0,0 +1,40 @@
+# Overview 
+
+`vets-website` has several parts that require ongoing maintenance and improvement. These platform parts are assigned a primary and a secondary engineer. 
+
+This document describes the responsibilities of each of these roles and is intended for VSP frontend engineers and their leadership group. 
+
+## Primary 
+
+- Accountable for the ongoing maintenance of their designated platform part 
+  - Expected to do around a day of unplanned maintenance per sprint- this does not include support time but should be documented in a ticket and draft PR
+  - Inform engineering lead and product manager if no maintenance has been performed for 2 consecutive sprints 
+- Assists product manager with backlog management 
+  - Understands the implementation of the platform part 
+  - Provides feasible improvement options 
+  - Understand the problem solved by the platform part
+  - Makes tradeoff decisions with product manager 
+- Provide consultation and support to users 
+- Consults and informs secondary engineer for all changes made to the platform part
+  - Secondary engineer is required approver on all changes when available 
+- Keeps documentation organized and up to date 
+
+## Secondary 
+
+- Responsible for the ongoing maintenance of their designated platform part 
+- Reviews and approves all changes made 
+- Familiar with backlog and ongoing changes 
+- Can provide support on platform part when primary is unavailable 
+
+# Assignments 
```

Comment:
docs that list out a bunch of individual names scare me (outdated quick), especially buried in a side doc --- wdyt about putting this somewhere more prominent where that's less likely (i.e. platform/engineering/frontend/README.md), or do you have another strategy for keeping this up-to-date?

Reply:
I don't think it belongs in the readme. It's mostly up to me to keep this updated since it's one of the primary ways I want to manage the FE group. 

We could just move the whole document into the VSP teams folder. I don't think the structure of these documents really matters though.

Reply:
In my vision of an ideal world, every document is linked to by at least one other. While this may not make sense to put in a README, if we had one, we could link to this and other important documents.

Reply:
Moved https://docs.google.com/spreadsheets/d/1loQMRW8zNUYZXzcBsIVVKWLtHugw6uiNer1HTWgs980/edit#gid=1377110857